File: SepWebshop/SepWebshop.Domain/Insurances/Insurance.cs
namespace SepWebshop.Domain.Insurances;

public sealed class Insurance
{
    public Guid Id { get; set; }
    public required string Name { get; set; }
    public string? Description { get; set; }
    public float PricePerDay { get; set; }
    public float DeductibleAmount { get; set; }
}

File: sepwebshop-frontend/src/app/models/interfaces/insurance.ts
export interface IInsuranceResponse {
  id: string;
  name: string;
  description: string;
  pricePerDay: number;
  deductibleAmount: number;
}

export interface IAddInsuranceRequest {
  name: string;
  description: string;
  pricePerDay: number;
  deductibleAmount: number;
}
